pkg/apis/core/validation/seed.go

```diff
@@ -239,6 +228,50 @@ func ValidateSeedSpec(seedSpec *core.SeedSpec, fldPath *field.Path, inTemplate b
 	return allErrs
 }
 
+func validateSeedBackup(seedBackup *core.SeedBackup, seedProviderType string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if seedBackup == nil {
+		return allErrs
+	}
+
+	if len(seedBackup.Provider) == 0 {
+		allErrs = append(allErrs, field.Required(fldPath.Child("provider"), "must provide a backup cloud provider name"))
+	}
+
+	if seedProviderType != seedBackup.Provider && (seedBackup.Region == nil || len(*seedBackup.Region) == 0) {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("region"), "", "region must be specified for if backup provider is different from provider used in `spec.cloud`"))
```

Comment:
What is `.spec.cloud`?

Reply:
😄 We need to find out, in master branch it is the same https://github.com/gardener/gardener/blob/3936598e7fee0e029b7bd93a88e5ca95cda81314/pkg/apis/core/validation/seed.go#L113

Reply:
:D maybe you can clean it up